val/src/avs_wakeup.c

```diff
@@ -42,6 +42,10 @@ val_wakeup_execute_tests(uint32_t level, uint32_t num_pe)
       }
   }
 
+  if (g_single_module != SINGLE_MODULE_SENTINEL && g_single_module != AVS_WAKEUP_TEST_NUM_BASE) {
+    return AVS_STATUS_SKIP;
```

Comment:
Add log print for consistency in log
val_print(AVS_PRINT_TEST, "      USER Override - Skipping all Wakeup tests (running only a single module)\n", 0);

Reply:
Fixed.